org.librarysimplified.audiobook.feedbooks/src/main/java/org/librarysimplified/audiobook/feedbooks/UnsupportedAlgorithmException.kt

```diff
@@ -0,0 +1,3 @@
+package org.librarysimplified.audiobook.feedbooks
+
+class UnsupportedAlgorithmException(message: String?) : Exception(message)
```

Comment:
In the future, you might consider consolidating all these exceptions into one file. This is very much a Kotlin-y thing though and what you have here is totally fine.

```
/* org.librarysimplified.audiobook.feedbooks.Exception.kt */

class UnknownIssuerException(message: String?) : Exception(message)
class UnsupportedAlgorithmException(message: String?) : Exception(message)
```

Reply:
Thanks -- I'm new to Kotlin, so I appreciate any tips on Kotlin-isms.